### PR TITLE
Split DeviceAgent and iOSDeviceManager tasks

### PR DESCRIPTION
**Pull request by AndreyMaslennikov**
In this PR we have split DeviceAgent and iOSDeviceManager rake tasks.
Also we have fixed incorrect logic that was in `device_agent:clean` task before

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -1,4 +1,5 @@
 require 'bundler'
+require 'pry'
```
Comment (ivan-nosar) [inline Rakefile]: Please delete this line

File: Rakefile [hunk 2]
```diff
@@ -19,104 +20,103 @@ begin
...
-  end
+def bin
+  @bin ||= File.join(device_agent_dir, "bin", "iOSDeviceManager")
+end
 
-  def bin
-    @bin ||= File.join(device_agent_dir, "bin", "iOSDeviceManager")
-  end
+def license
+  @license ||= File.join(device_agent_dir, "bin", "iOSDeviceManager.LICENSE")
```
Comment (ivan-nosar) [inline Rakefile]: This line sounds like "Take `iOSDeviceManager.LICENSE` from DeviceAgent folder". Could you please explain this behavior?

File: Rakefile (hunk 1 above)
Comment (AndreyMaslennikov) [reply Rakefile]: Done

File: Rakefile (hunk 2 above)
Comment (AndreyMaslennikov) [reply Rakefile]: Previously `:clean` task has not delete `iOSDeviceManager.LICENSE` file. I've added this def to delete it in `iOSDeviceManager:clean` task

File: Rakefile [hunk 3]
```diff
@@ -354,10 +309,104 @@ Failed to create the codesigning keychain:
...
   task :checkout do
     banner("Git Checkout")
-    checkout(bin)
-    checkout(frameworks_zip)
     checkout(app_zip)
     checkout(ipa_zip)
   end
 end
 
+namespace :iOSDeviceManager do
```
Comment (jmoody) [inline Rakefile]:
```suggestion
namespace :idm do
```

Comment (AndreyMaslennikov) [reply Rakefile]: Thanks.
`iOSDeviceManager` namespace was renamed to `idm`

**Pull request merged**